app/javascript/src/apis/registrations.js

```diff
@@ -0,0 +1,9 @@
+import axios from "axios";
+
+const updatePassword = payload => axios.put("my/password/update", payload);
+
+const registrationsApi = {
+  updatePassword,
+};
+
+export default registrationsApi;
```

Comment:
```suggestion
export default {
  updatePassword,
};
```

Reply:
@vinay0x 

This was intentional. It helps a lot with the intellisense. We don't have to manually import from the apis if we write like this. We could just type in `registrationsApi` and vscode will auto import for us.

This widely helps with faster writing of code with least amount of content switch. 

Requesting you to allow to keep it. Thanks.

Reply:
Few things that immediately come to my mind:
1) For the IntelliSense to work, user needs to be aware of the name of the exported object right? They've to know the name to type `registrationsApi`. 
2) Number of times we reuse the same API file is very less.
3) If everything isn't configured perfectly, IntelliSense will use relative path for auto import.

Anyways if you insist on keeping this change, please make the change in other places.

Reply:
I will make the changes everywhere.

- I thought we had a standard in place while writing Apis.
  * All apis reside in `apis/`
  * Naming of api file  = `pluralized-resource-name.js`. Eg:- `registrations.js`
  * Naming of the api object = `pluralizedResourceNameApi`. Eg:- `registrationsApi`
  * (Optional or Maximum try to) Try our best to name the actions within that api align with the actions associated with that resource. Eg:- `update, destroy, create etc`. 

If we don't have such a standard, then I suggest we can enforce it. Such a convention would help with easier debugging and easier coding when someone new starts with it. They only have to keep the `routes.rb` in mind. Rest they could just type without actually going into the `apis` folder and manually checking which file and object name should be used.

Reply:
Will take this up in https://github.com/bigbinary/wheel/issues/453

Reply:
Handle in this PR itself. It's a small change. The APIs in this repos follows it. @yedhink _a